tests/test_modeling_blenderbot.py

```diff
@@ -0,0 +1,484 @@
+            activation_function="gelu",
+            normalize_before=False,
+            static_position_embeddings=True,
+            dropout=0.1,
+        )
+        input_ids = torch.tensor(
+            [[49, 12, 38, 24, 13, 25, 10, 28, 37, 7, 44, 7, 2, 3]], dtype=torch.long, device=torch_device,
+        )
+        model = BlenderbotForConditionalGeneration(config)
+        model.eval()
+
+        # output from parlai model after copying the same blenderbot weight in parlai and setting a manual_seed
+        expected_logits = torch.tensor(
+            [
+                [
+                    [
+                        -1.0000e20,
+                        0.0000e00,
+                        2.6462e-02,
+                        9.7588e-02,
+                        -5.6271e-02,
+                        -1.1409e-01,
+                        -3.3294e-02,
+                        -1.0423e-01,
+                        -4.8363e-02,
+                        -1.2610e-01,
+                        -3.4125e-02,
+                        -2.9841e-02,
+                        8.6975e-02,
+                        2.5547e-02,
+                        2.0425e-03,
+                        -5.9153e-02,
+                        1.4392e-02,
+                        -1.4324e-02,
+                        1.2774e-01,
+                        -5.3284e-02,
+                        -1.5876e-02,
+                        9.1752e-02,
+                        -3.0166e-02,
+                        -3.1726e-02,
+                        9.9600e-02,
+                        1.0991e-01,
+                        8.0946e-03,
+                        3.5396e-03,
+                        -2.5164e-02,
+                        -4.0277e-02,
+                        -3.6360e-02,
+                        7.5158e-02,
+                        4.3379e-02,
+                        1.3465e-01,
+                        -1.3209e-01,
+                        -1.1706e-01,
+                        5.6180e-02,
+                        -3.6239e-02,
+                        6.6490e-02,
+                        4.9879e-02,
+                        1.0979e-02,
+                        -2.7895e-02,
+                        -8.4691e-02,
+                        4.5857e-02,
+                        2.0233e-02,
+                        9.2533e-02,
+                        -6.4260e-02,
+                        -5.0988e-02,
+                        -4.0852e-02,
+                        1.9867e-02,
+                    ]
+                ]
+            ]
+        )
+
+        decoder_inputs = torch.LongTensor([1]).expand(1, 1).to(torch_device)
+        logits = model(input_ids, decoder_input_ids=decoder_inputs)["logits"]
+        assert torch.allclose(expected_logits, logits, atol=1e-4)
+
+    @slow
+    def test_generation_from_short_input_same_as_parlai_3B(self):
+        model = BlenderbotForConditionalGeneration.from_pretrained("facebook/blenderbot-3B").to(torch_device)
```

Comment:
caching the model here as @sshleifer has done for the Bart tests would save quite some testing time here.